db/seeds/dummy_structures.rb

```diff
@@ -156,6 +156,21 @@
 end
 ParticipantProfileState.find_or_create_by!(participant_profile: profile)
 
+# populate change of circumstances
```

Comment:
I added some seeds to the feature branch that I think we can delete following this: https://github.com/DFE-Digital/early-careers-framework/pull/1826/files#diff-d5f7c5155ac6dc126bde99f03ae835fbe6d87591e2193e553d20657d2e8750fdR5

Reply:
Yes 👍 , perhaps when we come to merge that branch in we can remove them, this will at least get the correct things in place for everything else